File: VB/Demos/12Sql/EFCore.vb
Imports Microsoft.EntityFrameworkCore

Namespace Sql

  Public Class EFCore
    Private ReadOnly _connectionString As String

    Public Sub New(connectionString As String)
      _connectionString = connectionString
    End Sub

    Public Sub EFCoreSelect()

    End Sub

    Public Sub EFCoreUpdate()

    End Sub

    Public Sub EFCoreUpdateTransaction()

    End Sub

  End Class
End Namespace

File: CS/Demos/12Sql/EFCore.cs
using Microsoft.EntityFrameworkCore;

namespace Demos.Sql;

class EFCore
{
  private readonly string _connectionString;

  public EFCore(string connectionString) =>
    _connectionString = connectionString;

  public void EFCoreSelect()
  {
  }

  public void EFCoreUpdate()
  {
  }

  public void EFCoreUpdateTransaction()
  {
  }
}
